CMakeLists.txt

```diff
@@ -138,8 +303,10 @@ install (FILES src/pipeline.h DESTINATION include/nanomsg)
 install (FILES src/survey.h DESTINATION include/nanomsg)
 install (FILES src/bus.h DESTINATION include/nanomsg)
 
-install (TARGETS nanocat RUNTIME DESTINATION bin)
+if (BUILD_NANOCAT)
```

Comment:
@gdamore , I expect your commit to be the next on top of this PR -- would you fix this bug? Rather than `BUILD_NANOCAT` it simply needs to be `ENABLE_NANOCAT` -- or, whatever we decide the name of this option should be over on this discussion on #591